src/store/actions.js

```diff
@@ -617,29 +619,38 @@ export default {
 				if (cursor === undefined) {
 					throw new Error('Unified list has no tail')
 				}
+				const newestFirst = getters.getPreference('sort-order') === 'newest'
 				const nextLocalUnifiedEnvelopes = pipe(
 					findIndividualMailboxes(getters.getMailboxes, mailbox.specialRole),
 					map(getIndivisualLists(query)),
 					combineEnvelopeLists,
```

Comment:
```suggestion
					combineEnvelopeLists(getters.getPreference('sort-order')),
```

To fix https://github.com/nextcloud/mail/pull/8231#issuecomment-1802282236